lib/AST/Builtins.cpp

```diff
@@ -1089,16 +1089,15 @@ static ValueDecl *getAutoDiffApplyAssociatedFunction(
   auto *paramIndices = AutoDiffParameterIndicesBuilder(
       origFnTy, /*setAllParams*/ true).build(Context);
   // Generator for the resultant function type, i.e. the AD associated function.
-  BuiltinGenericSignatureBuilder::LambdaGenerator resultGen {
-    [=](BuiltinGenericSignatureBuilder &builder) -> Type {
-      auto assocFnTy = origFnTy->getAutoDiffAssociatedFunctionType(
-          paramIndices, /*resultIndex*/ 0, /*differentiationOrder*/ 1,
-          kind, /*lookupConformance*/ nullptr);
-      if (isMethod)
-        return assocFnTy->getResult()->castTo<AnyFunctionType>()->getResult();
-      return assocFnTy->getResult();
-    }
-  };
+  BuiltinGenericSignatureBuilder::LambdaGenerator resultGen{
```

Comment:
very minor: a space was deleted, if you care
```suggestion
  BuiltinGenericSignatureBuilder::LambdaGenerator resultGen {
```

Reply:
I'll leave it like this because clang-format put it this way